pkg/skaffold/docker/parse.go

```diff
@@ -185,11 +186,26 @@ func extractCopyCommands(nodes []*parser.Node, onlyLastImage bool, insecureRegis
 	for _, node := range nodes {
 		switch node.Value {
 		case command.From:
-			wd, err := WorkingDir(node.Next.Value, insecureRegistries)
-			if err != nil {
-				return nil, err
+			from := fromInstruction(node)
+			if from.as != "" {
+				// Stage names are case insensitive
+				stages[strings.ToLower(from.as)] = true
+			}
+
+			// If `from` references a previous stage, then the `workdir`
+			// was already changed.
```

Comment:
is this always true? can I not change the `workdir` in my dockerfile after I copy something from a previous stage?

```dockerfile
COPY --from=prev_stage /app/app.go . # we could be anywhere at this point
WORKDIR /workdir # this contains some script that operates on the file that I just copied
RUN ./script.sh
```

this example might be contrived but isn't it theoretically possible?

Reply:
Would you have a full Dockerfile that doesn’t pass the tests?

Reply:
hmm, I tested this Dockerfile and it seems to work. I guess I just got a little confused by this comment, disregard!